tools/util/include/cutlass/util/reference/host/tensor_fill.h

```diff
@@ -64,6 +64,18 @@ namespace host {
 
 namespace detail {
 
+/// Returns a value of the Gaussian distribution generated by the Box Muller method
+__forceinline__ 
```

Comment:
we don't need `__forceinline__` here. compiler will do it.  we dont use it explicitly in this file.

Reply:
I'll remove it